coalib/parsing/Globbing.py

```diff
@@ -388,13 +388,28 @@ def iglob(pattern):
         pat = os.path.normcase(pat)
 
         if pat.endswith(os.sep):
-            for name in _iglob(pat):
+            for name in _simple_glob(pat):
                 yield name
         else:
-            for name in _iglob(pat):
+            for name in _simple_glob(pat):
                 yield name.rstrip(os.sep)
 
 
+def iglob(pattern):
+    """
+    DEPRECATED. Do not use. Wraps glob.
+
```

Comment:
@sils Why not really mark as deprecated? By raising a deprecation warning. Or is there a decorator?